torch/_inductor/fx_passes/mkldnn_fusion.py

```diff
@@ -783,6 +783,12 @@ def is_linear_add_bias(match):
             add_node = match.output_node()
             linear_node = add_node.args[0]
             weight_meta = linear_node.args[1].meta.get("val")
+            if not hasattr(add_node.args[1], "meta"):
+                # May add an "int"
+                # We meet this issue while enabling bf32 for test_linear_unary
+                # the hardsimoid case will add "3" here and cannot check meta
+                # TODO: Further investigate on this issue
```

Comment:
Is it for the bias node or something else? Can you double check? Why didn't we get the problem in the past?

Reply:
It should be another input node for "add".
We did not meet this in the past because we only enabled and tested lp for mkldnn fusion. For lp case, there will be a "to_dtype" inserted by autocast and will not match this pattern.

I am trying to find a case to satisfy this pattern by raising a runtime error if this function will return True.
https://github.com/pytorch/pytorch/pull/127597

Reply:
Fixed in https://github.com/pytorch/pytorch/pull/127597